packages/hopli/src/faucet.rs

```diff
@@ -49,7 +49,7 @@ pub struct FaucetArgs {
         long,
         short = 't',
         value_parser = clap::value_parser!(f64),
-        default_value_t = 2000.0
```

Comment:
I would not change the defaults here since we don't know which processes rely on those. (e.g. in devops)